package.json

```diff
@@ -59,6 +59,8 @@
     "prismjs": "~1.29.0",
     "react-icons": "~4.10.1",
     "react-intl": "~6.4.4",
+    "rehype-autolink-headings": "^6.1.1",
```

Comment:
Please read our Dependency Pinning guidelines :) https://github.com/nodejs/nodejs.org/blob/main/DEPENDENCY_PINNING.md

These should have a `~` version range and not a `^` (both the dependencies you added, and after fixing them, don't forget to run `npm i` again to update the lockfile)